docs/getting-started/data-views-in-sec.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[data-views-in-sec]]
+= {data-sources-cap} in Elastic Security
+
+The active {data-source} determines what data appears throughout {es-sec} on pages with event or alert data.
+{data-sources-cap} are defined by the index patterns they include.
+Only data from the {es} {ref}/documents-indices.html[indices], {ref}/data-streams.html[data streams], or {ref}/alias.html[index aliases] specified in the active {data-source} will appear.
+
+IMPORTANT: Custom indices may not be included in the <<default-data-view-security, default {data-source}>>. Modify it or create a custom {data-source} to include custom indices.
+
+[discrete]
+== Switch to another {data-source}
+
+You can tell which {data-source} is active by clicking the {data-source-cap} button that appears at the upper right of {es-sec} pages that display data, such as Overview, Alerts, Timelines, or Hosts.
+To switch to another {data-source}, click **Choose {data-source}**, select one of the options, and click **Save**.
+
+image::images/dataview-select-gif.gif[gif of opening the data view selection menu]
```

Comment:
I think it's helpful to have an illustration here, but a static image would probably suffice -- all this is really showing is how to open a simple menu. A regular PNG with highlights could still illustrate the general location of the menu and what it looks like when opened, and it'd be easier to reproduce/crop/stitch in the future.

That said, the process that @stephmilovic mentions [below](https://github.com/elastic/security-docs/pull/2034/files#r888377300) is more complex, and so an animated GIF seems more appropriate to demonstrate that complexity.